src/main/java/lotto/lotto/LottoBundle.java

```diff
@@ -0,0 +1,99 @@
+package lotto.lotto;
+
+import lotto.purchase.PurchaseService;
+import lotto.util.LottoUtil;
+import lotto.util.UserOutput;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoBundle {
+    private List<Lotto> lottos = new ArrayList<>();
+    private Lotto winningLotto;
+    private int bonus;
+
+
+    /**
+     * lottoNum의 수만큼 새로운 로또 생성하기
+     */
+    public void createLottos(int lottoNum) {
+        for (int i = 0; i < lottoNum; i++) {
+            Lotto lotto = new Lotto(LottoUtil.createLotto());
+            lottos.add(lotto);
+        }
+    }
+
+    /**
+     * 당첨번호 저장하기
+     */
+    public void setWinningLotto(List<Integer> winningNumList) {
+        Lotto lotto = new Lotto(winningNumList);
+        this.winningLotto = lotto;
```

Comment:
```suggestion
        this.winningLotto = new Lotto(winningNumList);
```
재사용 가능성이 없는 경우 변수에 담지 않고 바로 사용하는게 더 좋은 것 같아요!